notebooks/adastra/data-owners/00-data-owners-provision-ubuntu.ipynb

```diff
@@ -127,67 +130,67 @@
    "id": "cceb29d4-5fe5-46f4-a4c6-bb63b0cd011b",
    "metadata": {},
    "source": [
-    "Once HAGrid has installed you can simply type `hagrid` to check it is working"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "897f767e-eec9-496e-b997-fe0a52629bad",
-   "metadata": {},
-   "source": [
-    "```\n",
+    "Once HAGrid has installed you can simply type `hagrid` on the terminal to check if it is working.\n",
+    "\n",
+    "```bash\n",
     "$ hagrid\n",
-    "```"
+    "```\n",
+    "\n",
+    "You should see the following table."
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "c59bf0a9-371a-40af-a4f6-c93d13df07a1",
+   "id": "6af9ca34-ede8-483a-bebb-2cbc55d8769d",
    "metadata": {},
    "source": [
-    "You should see the following"
+    "![hagrid_runs](img/hagrid_runs.png)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "6af9ca34-ede8-483a-bebb-2cbc55d8769d",
+   "id": "03be73eb",
    "metadata": {},
    "source": [
-    "![hagrid_runs](img/hagrid_runs.png)"
+    "HAGrid checks if all dependencies required for provisioning a Domain/Network node are installed. One can run: \n",
+    "```bash\n",
+    "$ hagrid --help\n",
+    "```\n",
+    "to see all the list of possible commands that can be run with the HAGrid cli tool.\n",
+    "\n",
+    "**Note**: We can see that *Docker* is not installed yet, but HAGrid will install it for us in the next step, when we provision our Domain node."
```

Comment:
Wonderful,Did not know that , hagrid also installs docker for us, I initiallly thought ,  it installs only Docker compose v2 if not present.